components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -8,7 +8,7 @@ param (
 
 # # License and Copyright
 # ```
-# Copyright: Copyright (c) 2015 Chef Software, Inc.
+# Copyright: Copyright (c) 2016-2017 Chef Software, Inc.
```

Comment:
The copyright notice should be the year the work first appeared. So this should just become

`Copyright (c) 2017`